src/components/OnyxProvider.tsx

```diff
@@ -6,7 +6,7 @@ import ComposeProviders from './ComposeProviders';
 // Set up any providers for individual keys. This should only be used in cases where many components will subscribe to
 // the same key (e.g. FlatList renderItem components)
 const [withNetwork, NetworkProvider, NetworkContext] = createOnyxContext(ONYXKEYS.NETWORK);
-const [withPersonalDetails, PersonalDetailsProvider] = createOnyxContext(ONYXKEYS.PERSONAL_DETAILS_LIST);
+const [withPersonalDetails, PersonalDetailsProvider, , usePersonalDetails] = createOnyxContext(ONYXKEYS.PERSONAL_DETAILS_LIST);
```

Comment:
100% intentional? just asking :P

Reply:
Haha, good catch! ^^ Edu asked the same [here](https://callstack-hq.slack.com/archives/C05R3TK2AAC/p1697704301775199?thread_ts=1697651988.834819&cid=C05R3TK2AAC). 

tl;dr; It is intentional as the context value is no needed and eslint did not allow `_` as a placeholder - so we stick to this - perfectly valid, although quirky syntax :D

Reply:
yeah, a valid syntax that looks like a poorly resolved conflict 😂 Thanks!

Reply:
just wanted to make sure, fine with me then :)

Reply:
❤️

Reply:
TIL 

FWIW I think it might be more obvious to use the eslint disable. This looks really weird to me 😄

Reply:
Yup the syntax is quite peculiar :D Wonder if there is an option in eslint to let us use `_` as a placeholder (if I am not mistaken such syntax is used for example in Python).

I am not really a big fan of adding eslint disable comments ^^

Reply:
Where there is a will there is [a Stack Overflow](https://stackoverflow.com/questions/64052318/how-to-disable-warn-about-some-unused-params-but-keep-typescript-eslint-no-un)!  🎉

Reply:
@marcaaron Let the people decide! https://expensify.slack.com/archives/C01GTK53T8Q/p1698394757794659 :D